{"name": "test_set_new_password", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00007FF6D6453E62+57250]\n\t(No symbol) [0x00007FF6D63CBC02]\n\t(No symbol) [0x00007FF6D629E0BB]\n\t(No symbol) [0x00007FF6D62D765E]\n\t(No symbol) [0x00007FF6D62D774C]\n\t(No symbol) [0x00007FF6D6310267]\n\t(No symbol) [0x00007FF6D62F67EF]\n\t(No symbol) [0x00007FF6D630E312]\n\t(No symbol) [0x00007FF6D62F6583]\n\t(No symbol) [0x00007FF6D62CCC71]\n\t(No symbol) [0x00007FF6D62CDE54]\n\tGetHandleVerifier [0x00007FF6D6704CF2+2879026]\n\tGetHandleVerifier [0x00007FF6D6756F30+3215472]\n\tGetHandleVerifier [0x00007FF6D674FD4F+3186319]\n\tGetHandleVerifier [0x00007FF6D64E5505+652869]\n\t(No symbol) [0x00007FF6D63D7518]\n\t(No symbol) [0x00007FF6D63D35F4]\n\t(No symbol) [0x00007FF6D63D36EC]\n\t(No symbol) [0x00007FF6D63C38E3]\n\tBaseThreadInitThunk [0x00007FFD63ED7344+20]\n\tRtlUserThreadStart [0x00007FFD659C26B1+33]", "trace": "restore_password_link_page = <pages.restore_password_link_page.RestorePasswordLinkPage object at 0x000001FB212A10C0>\n\n    def test_set_new_password(restore_password_link_page):\n        restore_password_link_page.set_new_password()\n        restore_password_link_page.repeat_new_password()\n        restore_password_link_page.click_to_save_btn_restore_password()\n>       restore_password_link_page.verify_invalid_token_error_message()\n\ntests\\test_restore_password_link.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages\\restore_password_link_page.py:35: in verify_invalid_token_error_message\n    actual_result = self.get_text_title_of_element(RestorePasswordLinkLocators.INVALID_TOKEN_ERROR_MESSAGE)\nbase\\base_object.py:56: in get_text_title_of_element\n    return self.is_visible(locator).text\nbase\\base_object.py:16: in is_visible\n    return self.wait.until(ec.visibility_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e3076cadcbc59d05806c7f8ca6964858\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x000001FB21293010>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \tGetHandleVerifier [0x00007FF6D6453E62+57250]\nE       \t(No symbol) [0x00007FF6D63CBC02]\nE       \t(No symbol) [0x00007FF6D629E0BB]\nE       \t(No symbol) [0x00007FF6D62D765E]\nE       \t(No symbol) [0x00007FF6D62D774C]\nE       \t(No symbol) [0x00007FF6D6310267]\nE       \t(No symbol) [0x00007FF6D62F67EF]\nE       \t(No symbol) [0x00007FF6D630E312]\nE       \t(No symbol) [0x00007FF6D62F6583]\nE       \t(No symbol) [0x00007FF6D62CCC71]\nE       \t(No symbol) [0x00007FF6D62CDE54]\nE       \tGetHandleVerifier [0x00007FF6D6704CF2+2879026]\nE       \tGetHandleVerifier [0x00007FF6D6756F30+3215472]\nE       \tGetHandleVerifier [0x00007FF6D674FD4F+3186319]\nE       \tGetHandleVerifier [0x00007FF6D64E5505+652869]\nE       \t(No symbol) [0x00007FF6D63D7518]\nE       \t(No symbol) [0x00007FF6D63D35F4]\nE       \t(No symbol) [0x00007FF6D63D36EC]\nE       \t(No symbol) [0x00007FF6D63C38E3]\nE       \tBaseThreadInitThunk [0x00007FFD63ED7344+20]\nE       \tRtlUserThreadStart [0x00007FFD659C26B1+33]\n\nvenv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException"}, "steps": [{"name": "set up new password", "status": "passed", "start": 1694672696314, "stop": 1694672697056}, {"name": "repeat new password", "status": "passed", "start": 1694672697056, "stop": 1694672698091}, {"name": "click to \"save\" btn", "status": "passed", "start": 1694672698092, "stop": 1694672698737}, {"name": "verify invalid token error message", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00007FF6D6453E62+57250]\n\t(No symbol) [0x00007FF6D63CBC02]\n\t(No symbol) [0x00007FF6D629E0BB]\n\t(No symbol) [0x00007FF6D62D765E]\n\t(No symbol) [0x00007FF6D62D774C]\n\t(No symbol) [0x00007FF6D6310267]\n\t(No symbol) [0x00007FF6D62F67EF]\n\t(No symbol) [0x00007FF6D630E312]\n\t(No symbol) [0x00007FF6D62F6583]\n\t(No symbol) [0x00007FF6D62CCC71]\n\t(No symbol) [0x00007FF6D62CDE54]\n\tGetHandleVerifier [0x00007FF6D6704CF2+2879026]\n\tGetHandleVerifier [0x00007FF6D6756F30+3215472]\n\tGetHandleVerifier [0x00007FF6D674FD4F+3186319]\n\tGetHandleVerifier [0x00007FF6D64E5505+652869]\n\t(No symbol) [0x00007FF6D63D7518]\n\t(No symbol) [0x00007FF6D63D35F4]\n\t(No symbol) [0x00007FF6D63D36EC]\n\t(No symbol) [0x00007FF6D63C38E3]\n\tBaseThreadInitThunk [0x00007FFD63ED7344+20]\n\tRtlUserThreadStart [0x00007FFD659C26B1+33]\n\n", "trace": "  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\pages\\restore_password_link_page.py\", line 35, in verify_invalid_token_error_message\n    actual_result = self.get_text_title_of_element(RestorePasswordLinkLocators.INVALID_TOKEN_ERROR_MESSAGE)\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\base\\base_object.py\", line 56, in get_text_title_of_element\n    return self.is_visible(locator).text\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\base\\base_object.py\", line 16, in is_visible\n    return self.wait.until(ec.visibility_of_element_located(locator))\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1694672698737, "stop": 1694672703987}], "start": 1694672696311, "stop": 1694672703993, "uuid": "3150617c-b42a-4115-b641-9082588d2bf3", "historyId": "40415a2bebe0f912186836b4a601f8a9", "testCaseId": "2f56d55f60fa301f66e7861bb2765108", "fullName": "tests.test_restore_password_link#test_set_new_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_restore_password_link"}, {"name": "host", "value": "DESKTOP-MNCDD93"}, {"name": "thread", "value": "4820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_restore_password_link"}]}